examples/07-EMIT/ComputeProtectionLevels.py.back

```diff
@@ -138,7 +138,8 @@ for band in bands:
 rev = emitapp.results.analyze()
 modeRx = econsts.tx_rx_mode().rx
 modeTx = econsts.tx_rx_mode().tx
-resultMode = econsts.result_type().powerAtRx
+modePower = econsts.result_type().powerAtRx
+modeEmi = econsts.result_type().emi
```

Comment:
I would just suggest to rename variables to follow Python snake_case convention.
```suggestion
mode_power = econsts.result_type().powerAtRx
mode_emi = econsts.result_type().emi
```